ihmc-perception/src/main/java/us/ihmc/perception/cuda/CUDAProgram.java

```diff
@@ -28,8 +28,8 @@
 public class CUDAProgram implements AutoCloseable
 {
    private static final String[] DEFAULT_OPTIONS = {"-arch=" + getComputeVersion(),  // Target fairly recent GPU architecture
-                                                    "--dopt=on",                     // Optimize code
-                                                    "-G"};                           // More code optimization
+                                                    /*"--dopt=on",                     // Optimize code
+                                                    "-G"*/};                           // More code optimization
```

Comment:
You should check whether it still works with these options enabled. Ideally we should be able to have these enabled